system/mpollux-digisign-client/mpollux-digisign-client.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="mpollux-digisign-client"
-VERSION="4.2.6b_8414"
+VERSION="4.3.4"
```

Comment:
You need to match the VERSION here with the one in SlackBuild file

Reply:
Fixed